lib/datasource/index.spec.ts

```diff
@@ -32,9 +32,8 @@ describe(getName(__filename), () => {
   });
   it('returns datasources', () => {
     expect(datasource.getDatasources()).toBeDefined();
-    expect(datasource.getDatasourceList()).toBeDefined();
   });
-  it('validates dataource', () => {
+  it('validates datsource', () => {
```

Comment:
```suggestion
  it('validates datasource', () => {
```
🤭

Reply:
Trading 1 spelling mistake for another :smile: